src/main/java/frc/robot/RobotContainer.java

```diff
@@ -72,23 +83,26 @@ public RobotContainer() {
   private void configureButtonBindings() {
     zeroGyro.onTrue(new InstantCommand(() -> s_Swerve.zeroHeading()));
     zeroOdometry.onTrue(new InstantCommand(() -> s_Swerve.setPose(new Pose2d(new Translation2d(15.3, 5.55), new Rotation2d(0)))));
-    aimToSpeaker.whileTrue(new TeleopAimSwerve(
-        s_Swerve,
-        () -> OIConstants.modifyMoveAxis(-driver.getRawAxis(translationAxis)), 
-        () -> OIConstants.modifyMoveAxis(-driver.getRawAxis(strafeAxis)), 
-        () -> s_Swerve.getHeadingToSpeaker()
-      )
+    
+    ampTemporary.whileTrue(new AmpCommand(
+      s_Swerve,
+      driver,
+      () -> driver.rightTrigger(.5, null).getAsBoolean())
+    );
+
+    intakeTemporary.whileTrue(new IntakeCommand(
+      s_Swerve, 
+      driver,  
+      override, 
+      () -> driver.rightTrigger(.5, null).getAsBoolean()).until(() -> s_IntakeSubsystem.noteInTransit())
```

Comment:
if you're already passing the whole controller (something i don't know if i can totally get behind, but it looks cleaner at least) you don't need to separately pass the override button which is already on that controller. at most i would pass the constant that points to the button if not at all.